org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/SignatureHelpHandlerTest.java

```diff
@@ -225,10 +225,32 @@ public void testSignatureHelp_constructor() throws JavaModelException {
 		ICompilationUnit cu = pack1.createCompilationUnit("E.java", buf.toString(), false, null);
 		SignatureHelp help = getSignatureHelp(cu, 3, 26);
 		assertNotNull(help);
-		assertEquals(5, help.getSignatures().size());
+		assertEquals(4, help.getSignatures().size());

```

Comment:
Changed from 5 to 4 because one of the `RuntimeException`'s constructor is not accessible actually.